sdks/java/io/pom.xml

```diff
@@ -81,11 +92,36 @@
               </execution>
             </executions>
           </plugin>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <version>${surefire-plugin.version}</version>
+            <configuration>
+              <skipTests>true</skipTests>
+            </configuration>
+          </plugin>
         </plugins>
       </build>
       <properties>
         <skipITs>false</skipITs>
       </properties>
     </profile>
+
+    <!-- this profile is for use with io-it-suite -->
+    <profile>
+      <id>forceDirectRunner</id>
+      <activation>
+        <property>
+          <name>forceDirectRunner</name>
+        </property>
+      </activation>
+      <properties>
+        <!-- These intentionally have a hanging equals sign so that an empty
+             string is passed to pkb. Passing "" will cause e.g. -P"" to
+             get added to the mvn command line -->
+        <pkbBeamRunnerProfile>-beam_runner_profile=</pkbBeamRunnerProfile>
```

Comment:
I don't love or totally understand this, but it is not an issue relevant to the maven configuration here.